nltk/test/unit/test_tokenize.py

```diff
@@ -21,3 +24,65 @@ def test_tweet_tokenizer(self):
         expected = [':', "Let's", 'test', 'these', 'words', ':', 'resumé',
                     'España', 'München', 'français']
         self.assertEqual(tokens, expected)
+
+    @skipIf(not os.environ.get('STANFORD_MODELS') or
+            not os.environ.get('STANFORD_SEGMENTER'),
+            "At least one of STANFORD_MODELS or STANFORD_SEGMENTER "
+            "environment variables is not set")
+    def test_stanford_segmenter(self):
+        """
+        The Stanford Word Segmenter for Arabic and Chinese
+        """
+        from nltk.tokenize.stanford_segmenter import StanfordSegmenter
+        from nltk.internals import find_file, find_jar, find_dir
+
+        _stanford_url = 'http://nlp.stanford.edu/software'
+        _JAR = 'stanford-segmenter.jar'
+        _SLF4J = 'slf4j-api.jar'
+
+        path_to_jar = find_jar(_JAR, None, env_vars=('STANFORD_SEGMENTER',),
```

Comment:
I would try to find the jar and catch all the Lookup exceptions, if nothing can be found then I would skip the test.